client/assets-proof/src/lib.rs

```diff
@@ -150,6 +153,7 @@ pub struct ParentVASPView {
     base_url: String,
     /// The number of child VASP accounts under this parent VASP account.
     num_children: u64,
+    diem_id_domains: Option<DiemIdDomains>,
```

Comment:
How about `Option<Vec<DiemIdVaspDomainIdentifier>>`?
From user's perspective, it is simpler and clear for `diem_id_domains: ["vasp-a"]`

Reply:
I might do a change after i'm done with events if it's possible to simplify the domain structure -- currently it's just mirroring the move resource structures